Components/Rating/rating.css

```diff
@@ -0,0 +1,69 @@
+.rating-container
+{
+    min-width: fit-content;
+    width: 100%;
+    height: fit-content;
+    padding: 4.8rem 1.6rem;
+    box-shadow: rgba(0, 0, 0, 0.02) 0px 1px 3px 0px, rgba(27, 31, 35, 0.15) 0px 0px 0px 1px;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-evenly;
+    align-items: center;
+    flex-wrap: nowrap;
+    gap: 3rem;
```

Comment:
height:fit-content, justify-content:center and then gap, maybe 2 of those 3 could be avoided (?)

Reply:
Done.

Reply:
@anasm11 @Naman-Saxena1   this can be changed later on, the toast is working fine, and it would work fine if you use it in your project. it is just netlify's sticky footer causing issues, you can migrate the website to other hosting services later.